allowlist.txt

```diff
@@ -0,0 +1,9 @@
+# Ensure that Django's SQLite backend module is included
+sqlite3/*
+lib-dynload/_sqlite3.so
+unittest/*
+wsgiref/*
+lib-dynload/_csv.so
+lib-dynload/_json.so
+# Django REST framework has a dependency on the Django test module
+django/test/*
```

Comment:
I take it this is P4A's allowlist, for use during copy?

Reply:
Yes - this allows things to be copied even if they are in the blocklist.